data/coaches.yml

```diff
@@ -0,0 +1,125 @@
+-
+  name: Silvana Precup
+  description: Secretary
+  image: silvana.jpg
```

Comment:
I see your point, and don't mind evolving this in the future.

However, keeping code we don't use right now usually creates unnecessary cruft, and makes it harder to understand for newcomers (or even us in 6 months). It's good practice to keep the codebase to the minimum necessary.

And we use git, so it's super easy to revert to your current commit, if or when we'll need it. ;)